content/en/docs/_index.md

```diff
@@ -20,23 +20,42 @@ Falco can detect and alert on any behavior that involves making [Linux system ca
 
 ## How Falco compares to other tools
 
-People often ask how Falco differs from [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux), [AppArmor](https://wiki.ubuntu.com/AppArmor), [Auditd](https://linux.die.net/man/8/auditd), and other tools related to Linux security policy. We wrote a [blog post](https://sysdig.com/blog/selinux-seccomp-falco-technical-discussion/) on the [Sysdig blog](https://sysdig.com/blog) comparing Falco to other tools.
+People often ask how Falco differs from [SELinux], [AppArmor], [Auditd], and other tools related to Linux security policy. We wrote a [blog post](https://sysdig.com/blog/selinux-seccomp-falco-technical-discussion/) on the [Sysdig blog](https://sysdig.com/blog) comparing Falco to other tools.
 
 ## How to use Falco
 
-Falco is deployed as a long-running daemon. You can install it as a [Debian](installation#debian)/[rpm](installation#rhel) package on a regular host or container host, you can deploy it as a [container](installation#docker), or you can build it [from source](source).
+Falco is deployed as a long-running daemon. There are several ways to run Falco:
 
-Falco is configured via (1) a [rules file](rules) that defines which behaviors and events to watch for and (2) a [general configuration file](configuration). Rules are expressed in a high-level, human-readable language. We've provided a sample rules file [`./rules/falco_rules.yaml`](https://github.com/falcosecurity/falco/blob/dev/rules/falco_rules.yaml) as a starting point—you can (and will likely want!) to adapt it to your environment.
+* Install it as a [Debian](installation#debian)/[rpm](installation#rhel) package on a regular host or container host
+* Deploy it as a [container](installation#docker)
+* Build it [from source](source)
 
-When developing rules, one helpful feature is Falco's ability to read trace files saved by the `scap` format. This allows you to "record" the offending behavior once and replay it with Falco as many times as needed while tweaking your rules.
+Falco configuration comes from two sources:
 
-Once deployed, Falco uses the kernel modules and eBPF probes to bring events to userspace. Falco watches for any events matching one of the conditions defined in the rule file. If a matching event occurs, a notification is written to the the configured output(s).
+1. A [rules file](rules) that defines which behaviors and events to watch for. Rules are expressed in a high-level, human-readable language. We've provided a sample rules file [`./rules/falco_rules.yaml`][falco_rules] as a starting point; you can—and will likely want to!—adapt it to your environment.
+1. A [general configuration file](configuration) that specifies things like logging behavior and time formats.
```

Comment:
A [general configuration file](configuration) that specifies parameters such as logging behavior and time formats.